### ISSUE TITLE
Default for FileBrowserField

### ISSUE THREAD

**Issue by herzog31** [sehmaschine/django-filebrowser#358]
Hi,

I try to select a default file for `FileBrowseField`. I tried the following approaches:

## Approach A
```
image = FileBrowseField(max_length=250, directory=image_folder, extensions=['.jpg'], blank=True, default='uploads/images/myFile.jpg')
```
I can create database migrations, but during migration I get the following error:
```
  File "/usr/local/lib/python3.6/site-packages/django/db/models/fields/__init__.py", line 762, in get_db_prep_value
    value = self.get_prep_value(value)
  File "/usr/local/lib/python3.6/site-packages/filebrowser/fields.py", line 104, in get_prep_value
    return value.path
AttributeError: 'str' object has no attribute 'path'
```
## Approach B
```
image = FileBrowseField(max_length=250, directory=image_folder, extensions=['.jpg'], blank=True, default=FileObject('uploads/images/myFile.jpg'))
```
With this approach I can't even create migrations. It returns the following error:
```
  File "/usr/local/lib/python3.6/site-packages/django/db/migrations/serializer.py", line 370, in serializer_factory
    "topics/migrations/#migration-serializing" % (value, get_docs_version())
ValueError: Cannot serialize: <FileObject: uploads/images/myFile.jpg>
There are some values Django cannot serialize into migration files.
For more, see https://docs.djangoproject.com/en/2.0/topics/migrations/#migration-serializing
```

## Environment
* Django-2.0.8
* django-filebrowser-3.10.1
* django-grappelli-2.11.1

**Comment by sehmaschine** [comment]
solved with stable/3.11.x

**Comment by DylanYoung** [comment]
I see the fix for A.  Was approach B fixed as well?
